app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -7,7 +7,9 @@
   describedby ||= nil
   controls ||= nil
   data ||= nil
-  autocomplete ||= nil
+  disable_form_submit_check = true if disable_form_submit_check
+  reveal_password = type == "password" && reveal_password ? true : false
+  autocomplete = reveal_password ? "off" : autocomplete ? autocomplete : nil
```

Comment:
I don't think that this double ternary is easily understandable - can it be split up?
```suggestion
  autocomplete ||= nil
  autocomplete = "off" if reveal_password
```

Reply:
Thanks, have changed.